src/docs/admin/references/properties/scheduler.properties

```diff
@@ -457,10 +457,10 @@ pa.scheduler.db.fetch.batch_size=50
 #-------------------------------------------------------
 
 # file containing variables which can be used in all workflows
-pa.scheduler.global.variables.configuration=config/scheduler/global_variables.properties
+pa.scheduler.global.variables.configuration=config/scheduler/global_variables.xml
 
-# file containing generic information which can be used in all workflows
-pa.scheduler.global.generic.info.configuration=config/scheduler/global_generic_info.properties
+# refresh period, in minutes, for the global variables configuration
+pa.scheduler.global.variables.refresh=10
 
```

Comment:
are these changes related to your PR ?

Reply:
No they appeared automatically after I updated the branch. Idk from where to be honest. I wanted to ask you about it.

Reply:
In the documentation, these changes are already committed 
https://trydev2.activeeon.com/doc/admin/ProActiveAdminGuide.html#_scheduler_properties
No problem to keep them

Reply:
Ok I'll keep them then.